msfrpc

```diff
@@ -41,7 +41,7 @@ opts = {
 }
 
 # Parse command line arguments.
-arguments.parse(ARGV) { |opt, idx, val|
+arguments.parse(ARGV) do |opt, idx, val|
```

Comment:
I'm a little sketchy on this, based only the fact that `do/end` blocks have a different precedence. Unless you have a rock solid test case I wouldn't reimplement this.

http://stackoverflow.com/questions/5587264/do-end-vs-curly-braces-for-blocks-in-ruby